api/config/routes.rb

```diff
@@ -131,7 +131,7 @@
         resource :cart, controller: :cart, only: %i[show create] do
           post   :add_item
           patch  :empty
-          # delete :destroy
+          patch  :delete
```

Comment:
```suggestion
          delete  :destroy
```

Reply:
@damianlegawiec I can't use `destroy` name for given action, because it's not registered properly by rails:
`rails routes | grep cart:`
`api_v2_storefront_cart DELETE                    /api/v2/storefront/cart(.:format)                                             spree/api/v2/storefront/cart#destroy {:format=>"json"}`
However when I change action name to `delete`, it's recorded properly:
`rails routes | grep cart:`
`delete_api_v2_storefront_cart DELETE                    /api/v2/storefront/cart/delete(.:format)                                      spree/api/v2/storefront/cart#delete {:format=>"json"}`

Do you allow to use `delete`?